### DISCUSSION TITLE
m4b to m4b conversion

### DISCUSSION THREAD

**Discussion by AVDemid** [yermak/AudioBookConverter#306] [Q&A]
I'm perfectly happy with chaptered single-file m4b audiobooks, but I want to make the files without music smaller by resampling them at 8 kHz and 8 kbps, which sound exactly the same to my ear. I can see the settings for this in the app and configure them accordingly but they don't apply to the conversion process when it starts. How do I do that?

**Comment by yermak** [comment]
Default behaviour of program not to reencode from aac too aac codec, which is used in m4b.
I believe that program does not compare source and target bitrates, as most of users would not check channels, sampling and bitrate of source file when reencoding from m4b to m4b, which would trigger explicit reencoding. 

So, i beleive this behaviour is simpler.

I was considering an option to add checkbox "force reencoding" or opposite "try to not reencode if possible" to make it work for you.

But this was will make UI more difficult for most of users.

If you badly need it - try to reencode original m4bs to mp3s and then build proper book in m4b with desired parameters.

If you have other thoughts how to make it better - please let me know.

**Comment by AVDemid** [comment]
Thank you, Yarick, for your prompt response. AVS Audio Converter provides m4b to m4a conversion that I need, and I use it now, but I'm always on the lookout for better solutions, so I took this (your?) converter out for a test drive after it was recommended here:  http://audiobookbay.nl/audio-books/aladdin-and-other-stories-audible-studios/
Aladdin and Other Stories - Audible Studios Audiobook Online Download, Free Audio Book Torrent, 162077<http://audiobookbay.nl/audio-books/aladdin-and-other-stories-audible-studios/>
Aladdin and Other Stories - Audible Studios audio book torrent free download, 162077. Shared by:kingabomination Written by Audible Studios Read by Tamsin Greig, Stephen Mangan Format: M4B Unabridged you can split this up yourself by book and chapter to mp3 format with yermak/AudioBookConverter. its a free opensource software on GitHub. Alternatively use Smart Audiobook player with .m4b format. This is my preferred method. Magical retellings of three classic fairy tales, [...]
audiobookbay.nl

Thank you for making such a good product available to us free of charge. Keep up the good work.

Best,

Alexander


________________________________
